pkg/codegen/nodejs/gen.go

```diff
@@ -722,7 +722,7 @@ func (mod *modContext) genResource(w io.Writer, r *schema.Resource) error {
 	// Emit the argument type for construction.
```

Comment:
Should we make the same change to the state inputs?  (a couple lines above this).

Reply:
Good catch! Done.